src/txkube/testing/_testcase.py

```diff
@@ -18,3 +18,18 @@ class TestCase(TesttoolsTestCase):
     def setUp(self):
         super(TestCase, self).setUp()
         self.useFixture(CaptureEliotLogs())
+
+    # expectThat and Hypothesis don't communicate well about when the
+    # test has failed.  Give them a little help.  These two Hypothesis
+    # hooks will check for a flag that testtools sets when it thinks
+    # the test has failed and turn it into something Hypothesis can
+    # recognize.
+    def setup_example(self):
+        try:
+            del self.force_failure
```

Comment:
:( -- maybe at very least a comment on why you're using del instead of getattr (I can guess why :/)